runtime/queries/gdscript/textobjects.scm

```diff
@@ -0,0 +1,14 @@
+
+(class_definition
+  (body) @class.inside) @class.around
+
+(function_definition
+  (body) @function.inside) @function.around
+
+(parameters
+  (typed_parameter 
+    (identifier) @parameter.inside
+  ) @parameter.around)
```

Comment:
We should probably cover untyped parameters as well

```scm
(parameters (identifier) @parameter.inside @parameter.around)
```

Reply:
thanks , I will Add this ...


```diff
(parameters 
  [
    (typed_parameter)
    (identifier)    
    (default_parameter)    
    (typed_default_parameter)  
  ] @parameter.inside @parameter.around)
```